Detectors/TRD/workflow/src/trd-tracking-workflow.cxx

```diff
@@ -42,14 +43,16 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 {
   // option allowing to set parameters
   std::vector<o2::framework::ConfigParamSpec> options{
-    {"disable-mc", o2::framework::VariantType::Bool, false, {"Disable MC labels"}},
-    {"disable-root-input", o2::framework::VariantType::Bool, false, {"disable root-files input readers"}},
-    {"disable-root-output", o2::framework::VariantType::Bool, false, {"disable root-files output writers"}},
-    {"enable-trackbased-calib", o2::framework::VariantType::Bool, false, {"enable calibration devices which are based on tracking output"}},
-    {"enable-qc", o2::framework::VariantType::Bool, false, {"enable tracking QC"}},
+    {"disable-mc", VariantType::Bool, false, {"Disable MC labels"}},
+    {"disable-pid", VariantType::Bool, false, {"Disable PID"}},
```

Comment:
could you change this here to `enable-pid` leaving the default `false` please? That way we don't need to update all steering scripts which generate workflows and add the `--disable-pid` flag there. When we want to have this in production we can then just add `--enable-pid`.